mywebsite/mywebsite/urls.py

```diff
@@ -1,8 +1,16 @@
-from django.contrib import admin 
+from django.contrib import admin
 from django.urls import path, include
+from django.contrib.sitemaps.views import sitemap
+from news.sitemaps import ArticleSitemap
+
+sitemaps = {
+    "articles": ArticleSitemap,
+}
 
 
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("news/", include("news.urls", namespace="news")),
+    path("sitemap.xml", sitemap, {"sitemaps": sitemaps},
+         name="django.contrib.sitemaps.views.sitemap",),
```

Comment:
name Это строка, так можно написать что-то менее громоздкое например sitemap, тогда и в шаблоне можно будет писать не django.contrib.sitemaps.views.sitemap, а просто и локанично - sitemap
```suggestion
         name="sitemap",),
```